apps/site/lib/site_web/views/trip_plan_view.ex

```diff
@@ -589,4 +601,22 @@ defmodule SiteWeb.TripPlanView do
       }
     )
   end
+
+  @spec get_highest_one_way_fare(TripPlan.Itinerary.t()) :: integer
+  def get_highest_one_way_fare(itinerary) do
+    itinerary.legs
+    |> Enum.filter(fn leg -> leg.description != "WALK" end)
```

Comment:
I think searching the description for "WALK" might be a bit brittle so let's get to the transit legs some other way. The [Leg.transit?](https://github.com/mbta/dotcom/blob/eaac9fdb9b32f0c55f64689ec23de01de283a513/apps/trip_plan/lib/trip_plan/leg.ex#L57) method might do the trick